src/OpenTelemetry.Instrumentation.Http/Implementation/HttpWebRequestActivitySource.netfx.cs

```diff
@@ -104,7 +98,15 @@ private static void AddRequestTagsAndInstrumentRequest(HttpWebRequest request, A
 
             if (activity.IsAllDataRequested)
             {
-                activity.SetCustomProperty(RequestCustomPropertyName, request);
+                try
+                {
+                    Options.Enrich?.Invoke(activity, "AddRequestTagsAndInstrumentRequest", request);
```

Comment:
We can name it ActivityStart

Reply:
should we rename method and string to OnStart, OnStop, OnException just like the others?

Reply:
method is fine. OnStartActivity, OnStopActivity, OnException are the names used in .net core, so lets keep it same here.

Reply:
updated!